exir/backend/test/test_lowered_backend_module.py

```diff
@@ -109,11 +107,7 @@ def forward(self, x):
             torch.allclose(model_outputs[0], expected_res, atol=1e-03, rtol=1e-03)
         )
 
-    @given(
-        unlift=st.booleans(),  # verify both lifted and unlifted graph
```

Comment:
Any specific reason we don't test lifted vs unlifted?

Reply:
these variables aren't actually being used, I'm not sure why it's not being tested, but it wasn't being tested to begin with, and pytest complains about that, so i'm just cleaning up here.

Reply:
Oh hmm, how did you find out they are not tested?

Reply:
unlift variable isn't used in the test body. pytest complains about that and says given is useless if that's the case.

Reply:
Ah looks like it was deleted in  https://github.com/pytorch/executorch/pull/3169

Reply:
Ah I understand now, it was used when exported program decided to lift all the params. It's not needed now params are all lifted anyway